molecule/default/tests/conftest.py

```diff
@@ -1,22 +1,23 @@
-"""PyTest Fixtures."""
-from __future__ import absolute_import
+# """PyTest Fixtures."""
```

Comment:
Better dropping files, than keeping commented code, it will still be in the git history if needed.